cmake/FindQuaZip.cmake

```diff
@@ -1,41 +1,36 @@
-# QUAZIP_FOUND               - QuaZip library was found
-# QUAZIP_INCLUDE_DIR         - Path to QuaZip include dir
-# QUAZIP_INCLUDE_DIRS        - Path to QuaZip and zlib include dir (combined from QUAZIP_INCLUDE_DIR + ZLIB_INCLUDE_DIR)
-# QUAZIP_LIBRARIES           - List of QuaZip libraries
-# QUAZIP_ZLIB_INCLUDE_DIR    - The include dir of zlib headers
+# QUAZIP_FOUND                   - QuaZip library was found
+# QUAZIP_INCLUDE_DIR             - Path to QuaZip include dir
+# QUAZIP_INCLUDE_DIRS            - Path to QuaZip and zlib include dir (combined from QUAZIP_INCLUDE_DIR + ZLIB_INCLUDE_DIR)
+# QUAZIP_LIBRARIES               - List of QuaZip libraries
+# QUAZIP_ZLIB_INCLUDE_DIR        - The include dir of zlib headers
 
-IF(QUAZIP_INCLUDE_DIRS AND QUAZIP_LIBRARIES)
-  # in cache already
-  SET(QUAZIP_FOUND TRUE)
-ELSE(QUAZIP_INCLUDE_DIRS AND QUAZIP_LIBRARIES)
-    IF(Qt5Core_FOUND)
+if(QUAZIP_INCLUDE_DIRS AND QUAZIP_LIBRARIES)
+    # in cache already
+    SET(QUAZIP_FOUND TRUE)
+else()
+    if(Qt5Core_FOUND AND NOT APPLE)
```

Comment:
` AND NOT APPLE` is not quite correct. quazip from Homebrew has `libquazip.dylib` and `quazip/quazip.h` as it's built with qmake, while quazip from MacPorts has `libquazip5.dylib` and `quazip5/quazip.h` as it's built with cmake. The difference between qmake/cmake is a upstream issue - https://github.com/stachenov/quazip/pull/28. Before the upstream issue is fixed, I suggest `FindQuaZip.cmake` in keepassxc should be able to find both `libquazip.dylib` and `libquazip5.dylib`.

Reply:
We do not officially support the Macports way, but I do agree on this. Thanks!

Reply:
Thanks for considering my suggestion! Here's a patch on top of the `hotfix/quazip-macos` branch that allows building KeePassXC with quazip from MacPorts:
```Diff
diff --git a/cmake/FindQuaZip.cmake b/cmake/FindQuaZip.cmake
index 66285fce..bd8fda1a 100644
--- a/cmake/FindQuaZip.cmake
+++ b/cmake/FindQuaZip.cmake
@@ -8,7 +8,7 @@ if(QUAZIP_INCLUDE_DIRS AND QUAZIP_LIBRARIES)
     # in cache already
     SET(QUAZIP_FOUND TRUE)
 else()
-    if(Qt5Core_FOUND AND NOT APPLE)
+    if(Qt5Core_FOUND)
         set(QUAZIP_LIB_VERSION_SUFFIX 5)
     endif()
     if(WIN32)
@@ -21,12 +21,12 @@ else()
         find_path(QUAZIP_ZLIB_INCLUDE_DIR NAMES zlib.h)
     else()
         find_library(QUAZIP_LIBRARIES
-            NAMES quazip${QUAZIP_LIB_VERSION_SUFFIX}
+            NAMES quazip quazip${QUAZIP_LIB_VERSION_SUFFIX}
             HINTS /usr/lib /usr/lib64 /usr/local/lib
         )
         find_path(QUAZIP_INCLUDE_DIR quazip.h
             HINTS /usr/include /usr/local/include
-            PATH_SUFFIXES quazip${QUAZIP_LIB_VERSION_SUFFIX}
+            PATH_SUFFIXES quazip quazip${QUAZIP_LIB_VERSION_SUFFIX}
         )
         find_path(QUAZIP_ZLIB_INCLUDE_DIR zlib.h HINTS /usr/include /usr/local/include)
     endif()
```

Reply:
Thanks. I submitted a different version, though, that also simplifies some other things.